src/main/java/org/team1540/robot2025/Constants.java

```diff
@@ -28,4 +28,23 @@ public enum Mode {
         /** Replaying from a log file. */
         REPLAY
     }
+
+    public static class IntakeConstants {
```

Comment:
This year, we want constants in their own files by subsystem to reduce merge conflicts
i.e. `robot2025/subsystems/intake/IntakeConstants.java`